wgpu-types/src/lib.rs

```diff
@@ -1303,6 +1358,49 @@ impl<L> TextureDescriptor<L> {
     }
 }
 
+/// Texture descriptor builder methods
+impl<'l> TextureDescriptor<Option<Cow<'l, str>>> {
```

Comment:
We should use a different name for the lifetime. How about `'a`?
The `'l` you have is probably meant for labels, but other descriptors have other stuff too, and we'd better use a single lifetime name consistently.